src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/CreateCaseTest.java

```diff
@@ -69,7 +70,7 @@ public void should_idempotently_create_case_from_valid_exception_record() throws
 
         // when
         // create case callback endpoint invoked first time
-        AboutToStartOrSubmitCallbackResponse callbackResponse = invokeCallbackEndpoint(exceptionRecord);
+        var callbackResponse = invokeCallbackEndpoint(exceptionRecord);
```

Comment:
do you think var is appropriate here ? invokeCallbackEndpoint(...) may not tell what would be the return type.
Although this is a test may not be needed to know that details ?

Reply:
discussed on slack